### ISSUE TITLE
TacO Icon Appears on 2nd row, covers party

### ISSUE THREAD

**Issue by BrianHVB** [BoyC/GW2TacO#144]
Since the update that adjusted the TacO icon position to account for the Wizard's vault, my icon now appears on a 2nd row instead of to the right of Black Lion 

![image](https://github.com/BoyC/GW2TacO/assets/17414903/e97dc3fd-4d4c-4735-8345-6479415259f3)

I'm running Widowed Full Screen @ 3440 x 1440

**Comment by BoyC** [comment]
That's definitely a wrong DPI setting issue, try running taco with the -forcedpiaware setting

**Comment by Technetium1** [comment]
I had something similar happen after opening an old game that can temporarily alter Windows scale factors (TF2). The same happens with BlishHUD. A reboot was enough to get it back to normal, though. The problem for me is having one display at a different scale factor, because it doesn't seem to happen when they're all the same, or with only one display.

**Comment by BrianHVB** [comment]
@BoyC No difference when running with that option,

**Comment by DanDecrypted** [comment]
@BoyC I don't think it is a wrong DPI setting issue. [This Commit](https://github.com/BoyC/GW2TacO/commit/023c432200ba8b48162de5a1ef61d097b9e93436) seems very intentional to move the taco button to where it is in the above screenshot.

**Comment by BoyC** [comment]
It was intentional, however if you look at the screenshot it's obvious that the TacO icon is oversized compared to the rest of the ui icons, which is either a DPI scaling issue usually or TacO fails to pick the correct css for the ui size based on what's supplied over the mumble link connection.
I actually don't have a pc right now (RMA hell) to check this out at the moment, but will when I can.
